tests/roles/cinder_adoption/tasks/main.yaml

```diff
@@ -14,6 +14,13 @@
   retries: 60
   delay: 2
 
+- name: Get cinder-api pod name
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc get pods -l component=cinder-api -o custom-columns="NAME:.metadata.name" --no-headers | head -1
+  register: cinder_api_pod
+
 - name: check that Cinder is reachable and its endpoints are defined
```

Comment:
we should have a `when` condition to make sure `cinder_api_pod.stdout | length`  is > 0, but I assume if we reach this point we already successfully passed L7 and `condition=Ready` ensures cinder-api(s) are up and running.

works for me.